.github/workflows/central-release-orchestrator.yml

```
@@ -192,18 +194,18 @@ jobs:
+            core.setOutput("source_sha",        sourceSha || "");
+            core.setOutput("release_label",     releaseLabel);
+            core.setOutput("release_level",     releaseLevel);
             core.setOutput("deploy_labels_json", JSON.stringify(deployLabels));
 
   create-tag:
     runs-on: ubuntu-latest
     needs: [resolve-context]
     outputs:
       version_tag: ${{ steps.bump.outputs.version_tag }}
-      target_sha: ${{ steps.bump.outputs.target_sha }}
+      target_sha:  ${{ steps.bump.outputs.target_sha }}
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)